src/components/common/modal/ModalOverlay.js

```diff
@@ -21,7 +21,7 @@ const getStylesFromProps = ({ theme }) => ({
     alignSelf: 'flex-start',
     backgroundColor: theme.modals.overlayBackgroundColor,
     height: height,
-    width: getScreenWidth(),
+    width: getOriginalScreenWidth(),
```

Comment:
this will probably not work on web, the screen width is very large

Reply:
use 100% maybe

Reply:
100%, it works

![Screenshot_20200124-111131_Chrome](https://user-images.githubusercontent.com/8187382/73057566-7e1df680-3e9a-11ea-8cc7-ffe66b18bcbb.jpg)
![image](https://user-images.githubusercontent.com/8187382/73057747-f5538a80-3e9a-11ea-9603-18c2d04cf3e2.png)